projects/1.16/assets/relics-of-gaming/rogblades/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+    "itemGroup.rog_mod":"游戏遗物",
+ 
+   "rogblades.name": "游戏遗物",
+   "rogblades.desc": "欢迎来到游戏遗物！",
+ 
+   "item.slashblade.ex.beastlord": "遗物「兽王」",
+   "item.slashblade.ex.beastlord.desc": "遗物「兽王」",
+   "item.slashblade.ex.beastlord.material": "刀组件「兽王」",
+ 
+   "item.slashblade.ex.virtuous": "遗物「贞洁」",

```

```suggestion
   "item.slashblade.ex.virtuous": "纯白契约",
```
同上，来自尼尔系列
刀名原neta可以查看curseforge简介，这类问题请在翻译文件中全部查明并修改
对于此类问题不再指出